packages/open-ui/src/components/Modal/CloseButton.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import classNames from 'classnames';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTimes } from '@fortawesome/free-solid-svg-icons';
+
+import styles from './CloseButton.module.scss';
+import { IntentOption } from 'interfaces/IntentOption';
+import { toSentenceCase } from 'utils/strings/to-sentence-case';
+
+interface CloseButtonProps {
+  close: () => void;
+  intent: IntentOption;
+}
+
+export const CloseButton: React.FC<CloseButtonProps> = ({ close, intent }) => {
```

Comment:
@remotealex why we need to recreate the styles for the closeButton under the Modal CloseButton ?

Reply:
From the original `<Button>`? It's different enough to have it's own separate file IMO